SpatialGDK/Documentation/content/get-started/tutorial.md

```diff
@@ -2,9 +2,9 @@
 
 ### What will be covered?
 
-In this tutorial you’ll implement cross server remote procedure calls (RPCs) in a simple third person shooter. The end result will be a multiplayer, cloud-hosted Unreal game running across multiple [server-workers]({{urlRoot}}/content/glossary#inspector) that players can seamlessly move between and shoot across.
+In this tutorial you’ll implement cross server remote procedure calls (RPCs) in a simple third person shooter. The end result will be a multiplayer, cloud-hosted Unreal game running across multiple [server-workers]({{urlRoot}}/content/glossary#inspector) that players can seamlessly move between and shoot across. It will look something like this:
 
-The exercise demonstrates that the workflows and iteration speed you’re used to as an Unreal developer are almost entirely unaltered by the GDK: it’s just like regular Unreal!
+![]({{assetRoot}}assets/tutorial/cross-server.gif)
```

Comment:
No more calling out the "like Unreal" claim?

Reply:
I removed this because I've called it out below. I can put it back in if you'd like though.

Reply:
Yeah I think we can call it twice in one doc.

Reply:
okay, will reinstate :)